### PR TITLE
test: extended: avoid using docker directly

### PR DESCRIPTION
**Pull request by runcom**
This test fails when running against CRI-O (https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_conformance_crio/40/console) because it uses docker.

Modify the test to check groups in container instead.

ref https://github.com/openshift/origin/issues/16403
@smarterclayton PTAL

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
can we also trigger tests with CRI-O on this /cc @kargakis

**Comment by runcom** [comment]
> /test crio

we need this though https://github.com/openshift/aos-cd-jobs/pull/640

**Comment by runcom** [comment]
> ci/openshift-jenkins/extended_conformance_crio

@kargakis , failure in crio (not sure it's related to crio, is it?)
```
"msg": "No package matching 'origin-3.7.0-0.alpha.1.452.62b50ee' found available, installed or updated", 
```

File: test/extended/security/supplemental_groups.go [hunk 1]
```diff
@@ -3,30 +3,28 @@ package security
...
 	e2e "k8s.io/kubernetes/test/e2e/framework"
-
-	testutil "github.com/openshift/origin/test/util"
 )
 
 var _ = g.Describe("[security] supplemental groups", func() {
 	defer g.GinkgoRecover()
 
+	const (
+		supplementaGroupsPod = "supplemental-groups"
```
Comment (smarterclayton) [inline test/extended/security/supplemental_groups.go]: typo

**Comment by smarterclayton** [comment]
One minor nit but otherwise looks good.

**Comment by runcom** [comment]
@kargakis @stevekuznetsov I suspect the CRI-O test fails because we're not building any release RPM in the crio job file, we may need something like this in the CRI-O job? https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/test_cases/test_branch_origin_extended_conformance_install.yml#L11-L35

**Comment by stevekuznetsov** [comment]
Yeah @kargakis the pull request job shouldn't be on the `build` image stage and needs to build itself.

**Comment by runcom** [comment]
@smarterclayton the 11 failures in crio tests are known and tracked by #16403 - this is good to merge as it's actually fixing one the failing tests in #16403

**Comment by runcom** [comment]
@smarterclayton can you guys merge this? the test is passing but crio CI is failing because of other tests failing.

**Pull request merged**